src/pages/SinglePlaylist.js

```diff
@@ -0,0 +1,23 @@
+import { Fragment } from 'react';
+import './SinglePlaylist.css';
+import { Header, VideoCard } from '../components';
+
+const SinglePlaylist = props => {
+  return (
+    <Fragment>
+      <Header />
+      <main className="main-single-playlist">
+        <div className="flex space-between align-center">
+          <h1>My Playlist</h1>
+          <button className="btn error rounded-edge">Delete Playlist</button>
+        </div>
+        <div className="videos-container">
+          {Array.from({ length: 4 }).map(el => (
+            <VideoCard dismissBtn={true} />
+          ))}
+        </div>
+      </main>
+    </Fragment>
```

Comment:
as you are not using any of the features of fragments instead of using this component just use empty tag "<></>"

Reply:
Thank you for pointing that out. I'll keep it in mind.